.gitignore

```diff
@@ -17,5 +17,4 @@ distribute-*.tar.gz
 up.bat
 anthrocon_plugin/
 add_table_extras.sql
-alembic*
```

Comment:
Huh, I could have sworn it DID work that way.  Weird.  I'm pretty sure Gitlab does work that way (an open source clone of Github which we use where I work).  That's a shame.